src/core/subscan.js

```
@@ -20,9 +20,5 @@ export function fetchTransactions({address, page = 0, row = 50}) {
         ...res.data,
         hasNextPage: (page + 1) * row < res.data.count,
       };
-    })
-    .catch(err => {
```

we're good to let the error escape?

---

Actually I removed this in the latest commit I think

---

We should propagate the error

---

yeah, the catch is gone and I was just making sure that allowing the error to bubble out now wasn't going to cause other problems.